pi_server/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.9

```

Comment:
I suggest you use `balenalib/raspberry-pi-python3.10-buster` instead.

Reply:
The version of python on the raspberry server is: Python 3.7.3 (default, Jan 22 2021, 20:04:44) 

https://github.com/balena-io-library/base-images/blob/master/balena-base-images/python/raspberrypi3/debian/buster/3.7.13/build/Dockerfile

I guess there are some things I should not touch in this file, I just need to replace FROM 
@Saxamos

Reply:
Use it with the right python version then :)